code/modules/research/xenobiology/crossbreeding/warping.dm

```diff
@@ -618,21 +617,15 @@ GLOBAL_DATUM(blue_storage, /obj/item/storage/backpack/holding/bluespace)
 		/obj/effect/mob_spawn/human/ash_walker,
 		/obj/effect/spawner/lootdrop/three_course_meal,
 		/mob/living/simple_animal/pet/dog/corgi/puppy/void,
-		/obj/structure/closet/crate/necropolis,
-		/obj/item/grenade/gluon,
 		/obj/item/card/emagfake,
 		/obj/item/gun/ballistic/revolver/reverse,
-		/obj/item/flashlight/flashdark,
```

Comment:
This one could stay, it's not harmful and you can make glowshrooms with same effect already.